tools/pipelines/test-dds-stress.yml

```diff
@@ -24,7 +24,8 @@ variables:
 - name: testWorkspace
   value: $(Pipeline.Workspace)/test
 # This is a test pipeline, not a build one, so we don't need to run CodeQL tasks
-- name: DisableCodeQL
+# Note that using this variable only works for non-1ES-template pipelines.
+- name: Codeql.SkipTaskAutoInjection
```

Comment:
Nit: can we link to the documentation for this parameter?

Reply:
Nit: I’m know whether it’s a test pipeline to determine if the pipeline is using a non-1ES template, but I think that might not be very obvious to team members who don’t have the context. Any other way for developer to know this is a non-1ES-template pipelines or not?